src/components/observations/ObservationComponent.tsx

```
@@ -116,19 +114,15 @@ const ObservationComponent: React.FC<QuestionnaireAppProps> = ({ observationQues
   };
 
   const receiveUploadData = (event: any) => {
-    console.log('event', event);
+ 
 
     if (event.data && event.data.file) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)